src/components/TodoCard/TodoCard.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+//style
+import './TodoCard.css'
+
+function TodoCard({ todo, ...props }) {
+    return (
+        <div className='todo-card'>
+            <div>ID : {todo?.id}</div>
+            <p className='todo-card--text'>{todo?.text}</p>
+            <div>Is important : {todo?.isImportant.toString()}</div>
+            <p className='todo-card--deeadline'>Deadline : {todo?.deadline}</p>
+            <div>Is Fineshed : {todo?.isFinshed.toString()}</div>
+        </div>
+    );
+}
+
+export default TodoCard;
```

Comment:
use 1 empty line in the end of the files. It will make more clear diffs in GIT while opening pull requests